dbms/src/Dictionaries/LibraryDictionarySource.cpp

```diff
@@ -137,7 +138,15 @@ LibraryDictionarySource::LibraryDictionarySource(const LibraryDictionarySource &
     , library{other.library}
     , description{other.description}
     , settings{other.settings}
+    , lib_data{other.lib_data}
+{
+}
+
+LibraryDictionarySource::~LibraryDictionarySource()
 {
+    auto fptr = library->tryGet<void * (*)(void *)>("ClickHouseDictionary_v2_libDelete");
```

Comment:
Inconsistent names: allocate vs. delete.
Better will be allocate and free (or deallocate) if we only allocating memory.
Or new/create/construct and delete/destroy/destruct if we also perform some initializations.